#USAGE parameter: value # comment (note that you cannot use tabs, use spaces instead)
ackermann:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 20

  left_rear_wheel_joint_controller:
    type: effort_controllers/JointVelocityController
    joint: left_rear_wheel_joint
    pid: {p: 1.0, i: 1.0, d: 0.0}

  right_rear_wheel_joint_controller:
    type: effort_controllers/JointVelocityController
    joint: right_rear_wheel_joint
    pid: {p: 1.0, i: 1.0, d: 0.0}

  left_steer_block_controller:
    type: effort_controllers/JointPositionController
    joint: left_steer_block_joint
    pid: {p: 1.0, i: 1.0, d: 0.1}

  right_steer_block_controller:
    type: effort_controllers/JointPositionController
    joint: right_steer_block_joint
    pid: {p: 1.0, i: 1.0, d: 0.1}




